scripts/SQLSERVER_Migration.sql

```diff
@@ -0,0 +1,46 @@
+DROP TABLE IF EXISTS [dbo].[TagTable];
```

Comment:
Just a general thought here - doing something destructive like dropping the TagTable... It might be better to fail the operation altogether and force the user to decide what to do in the event that they try to run this script twice. We can move the destructive / cleanup bits to a separate script that they can run explicitly that way. Buuuuuut for the sake of getting something into the repo that we can test now? I think this looks like it checks all the boxes.